src/server/collections/users-db.js

```diff
@@ -19,13 +20,23 @@ if (password !== undefined) {
 let mongodb = require('mongodb');
 let MongoClient = mongodb.MongoClient;
 
-function notFind(res, db, sendContent) {
+function createToken(userinfo, userAgent, callback) {
+  callback(tokensDb.createToken(userinfo._id, userAgent));
+}
+
+function notFind(req, res, db, sendContent) {
```

Comment:
this is notFound instead of notFind btw notFound is not a good choice neither...